### PR TITLE
Adding images to collections

### PR DESCRIPTION
**Pull request by gaizler**

### CODE REVIEW

File: PhotoService/ClientApp/src/services/profile.service.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ class ProfileService {
         .catch((err) => {throw err.response.data})
     }
 
-    getUserCollections(username) {
-        return axios.get(API_URL+'collections', {params: {username}})
+    getUserCollections(username,publicOnly) {
+        return axios.get(API_URL+`collections/?username=${username}&publicOnly=${publicOnly}`)
```
Comment (VitaliiDavydov) [inline PhotoService/ClientApp/src/services/profile.service.js]: better to use params
```suggestion
        return axios.get(API_URL+'collections', {params: {username, publicOnly}})
```

File: PhotoService/ClientApp/src/services/image.service.js [hunk 2]
```diff
@@ -37,6 +37,17 @@ class ImageService{
         });
     }
 
+    getImagesByColection(userName,collectionName){
+        return axios.get(API_URL + `/collection/${userName}?collectionName=${collectionName}`, {
```
Comment (vitaliy-davidov) [inline PhotoService/ClientApp/src/services/image.service.js:41]: better to use params

Comment (VitaliiDavydov) [inline PhotoService/ClientApp/src/services/image.service.js:41]:
```suggestion
        return axios.get(API_URL + '/collection', { params: {userName, collectionName},
```

File: PhotoService/ClientApp/src/components/profile-component/collections-component/Collections.js [hunk 3]
```diff
@@ -4,27 +4,29 @@ import CollectionItem from './collection-item-component/CollectionItem';
...
 
 const Collections = ({username}) => {
     const [collections, setCollections] = useState([]);
     const [showDialog, setShowDialog] = useState(false);
     const navigate = useNavigate();
 
-    const handleCollectionClick = (id) => {
-        navigate(`./collections/view/${id}`);
+    const handleCollectionClick = (name) => {
+        navigate(`./collections/?collectionName=${name}`);
```
Comment (vitaliy-davidov) [inline PhotoService/ClientApp/src/components/profile-component/collections-component/Collections.js:15]: better to use params
```suggestion
        navigate('./collections', { params: {collectionName: name}});
```

**Review by VitaliiDavydov — COMMENTED** [review]
looks good

**Pull request merged**